olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```
@@ -149,7 +149,7 @@ spec:
     - Support for managing Knative Eventing entities: Sources, Brokers, Triggers, Channels and Subscriptions
     - A kubectl-like plugin architecture to extend the built-in functionality
     - Easy integration of Knative into Tekton pipelines by using `kn` in a Tekton task
-    - Support for creating, building and deploying serverless functions for multiple runtimes (Node, Quarkus, Golang)
+    - Support for creating, building and deploying Knative Functions for multiple runtimes (Node, Quarkus, Golang)
```

Quarkus is the only "supported" runtime. How about this?
```suggestion
    - Create, build and deploy Knative Functions for multiple runtimes, including Node.js, Quarkus, and Go.
```